backend/de.metas.handlingunits.base/src/main/sql/postgresql/system/5713960_gh16902_HandlingUnitRestController_bySerialNo_onlyHUStatuses.sql

```diff
@@ -0,0 +1,4 @@
+-- 2023-12-18T17:00:21.778Z
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,Description,EntityType,IsActive,Name,Updated,UpdatedBy,Value) VALUES (0,0,541679,'S',TO_TIMESTAMP('2023-12-18 19:00:21.516','YYYY-MM-DD HH24:MI:SS.US'),100,'','de.metas.handlingunits','Y','de.metas.handlingunits.rest_api.bySerialNo.onlyHUStatuses',TO_TIMESTAMP('2023-12-18 19:00:21.516','YYYY-MM-DD HH24:MI:SS.US'),100,'A')
```

Comment:
i think this needs a docu
- where is it used (which API-Endpoint?)
- what's the systax - are spaces allowed?
- what are the possible HU-statuses?